RevenueCat/Scripts/PurchaserInfo.cs

```diff
@@ -24,9 +24,12 @@ public class PurchaserInfo
         public DateTime FirstSeen;
         public string OriginalAppUserId;
         public DateTime RequestDate;
+        [CanBeNull] public DateTime OriginalPurchaseDate;
```

Comment:
I believe this should be `DateTime?` instead of `CanBeNull` because `CanBeNull` indicates that the pointer can be null and `DateTime?` wraps it in an `Optional`, like `LatestExpirationDate`. I always get confused with this though